src/Common.OData.ApiExplorer/AspNet.OData/DefaultModelTypeBuilder.cs

```diff
@@ -311,6 +315,11 @@ static ModuleBuilder CreateModuleForApiVersion( ApiVersion apiVersion )
             return assemblyBuilder.DefineDynamicModule( "<module>" );
         }
 
+        static string GetNameFromAttribute( MemberInfo property )
+        {
+            return property.GetCustomAttribute<System.Runtime.Serialization.DataMemberAttribute>()?.Name;
+        }
```

Comment:
This can be removed as we want to rely solely on what the EDM says. It already supports consuming this information, so we'll rely on that behavior.

Reply:
Removed